cheqd/versions.json

```diff
@@ -246,6 +246,65 @@
         "type": "go",
         "version": "v7.4.0"
       }
+    },
+    {
+      "name": "v3.1.9",
+      "recommended_version": "v3.1.9",
+      "compatible_versions": [
+        "v3.1.9"
+      ],
+      "proposal": 64,
+      "height": 18394700,
+      "binaries": {
+        "linux/amd64": "https://github.com/cheqd/cheqd-node/releases/download/v3.1.9/cheqd-noded-3.1.9-linux-amd64.tar.gz",
+        "linux/arm64": "https://github.com/cheqd/cheqd-node/releases/download/v3.1.9/cheqd-noded-3.1.9-linux-arm64.tar.gz",
+        "darwin/amd64": "https://github.com/cheqd/cheqd-node/releases/download/v3.1.9/cheqd-noded-3.1.9-darwin-amd64.tar.gz",
+        "darwin/arm64": "https://github.com/cheqd/cheqd-node/releases/download/v3.1.9/cheqd-noded-3.1.9-darwin-arm64.tar.gz"
+      },
+      "previous_version_name": "v3.1.5",
+      "next_version_name": "v4",
+      "sdk": {
+        "type": "cosmos",
+        "repo": "https://github.com/cheqd/cosmos-sdk",
+        "version": "v0.47.10",
+        "tag": "v0.47.10-height-mismatch"
+      },
+      "ibc": {
+        "type": "go",
+        "version": "v7.10.0"
+      }
+    },
+    {
+      "name": "v4",
+      "recommended_version": "v4.1.1",
+      "compatible_versions": [
+        "v4.1.0",
+        "v4.1.1"
+      ],
+      "consensus": {
+        "type": "cometbft",
+        "version": "0.38.12"
+      },
+      "proposal": 65,
+      "height": 20206500,
+      "binaries": {
+        "linux/amd64": "https://github.com/cheqd/cheqd-node/releases/download/v4.1.1/cheqd-noded-4.1.1-linux-amd64.tar.gz",
+        "linux/arm64": "https://github.com/cheqd/cheqd-node/releases/download/v4.1.1/cheqd-noded-4.1.1-linux-arm64.tar.gz",
+        "darwin/amd64": "https://github.com/cheqd/cheqd-node/releases/download/v4.1.1/cheqd-noded-4.1.1-darwin-amd64.tar.gz",
+        "darwin/arm64": "https://github.com/cheqd/cheqd-node/releases/download/v4.1.1/cheqd-noded-4.1.1-darwin-arm64.tar.gz"
+      },
+      "previous_version_name": "v3.1.9",
+      "next_version_name": "",
+      "sdk": {
+        "type": "cosmos",
+        "repo": "https://github.com/cheqd/cosmos-sdk",
+        "version": "v0.50.14",
+        "tag": "v0.50.14-height-mismatch-iavl"
+      },
+      "ibc": {
+        "type": "go",
+        "version": "v8.7.0"
```

Comment:
Path: ibc.version, Reason: Value mismatch, In "codebase": v7.4.0, In currentVersion: v8.7.0

Please update the ibc field in the chain.json file to the same value.

Reply:
Updated to proper values.